.buildscript/update_changelog.swift

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/swift 
+
+import Foundation
+
+let contents = try! String(contentsOfFile: "CHANGELOG.md", encoding: .utf8)
+
+var comps = contents.components(separatedBy: "\n")
+
+let workflowVersion = ProcessInfo.processInfo.environment["WORKFLOW_VERSION"]!
+let kotlinChangelog = ProcessInfo.processInfo.environment["KOTLIN_CHANGELOG"]!
+let swiftChangelog =  ProcessInfo.processInfo.environment["SWIFT_CHANGELOG"]!
+
+let dateFormatter = DateFormatter()
+dateFormatter.dateFormat = "YYYY-MM-dd"
+
+let newChangelog = """
+
+## Version \(workflowVersion)
+
+_\(dateFormatter.string(from: Date()))_
+
+### Kotlin
+
+\(kotlinChangelog)
+
+### Swift
+
+\(swiftChangelog)
+"""
+
+comps.insert(newChangelog, at: 2)
+
+try! comps.joined(separator: "\n").write(toFile: "CHANGELOG.md", atomically: true, encoding: .utf8)
```

Comment:
What does `atomically` do and why is it necessary?

Reply:
`atomically` ensures a file either gets written successfully or fails without modifying (if app crashes, memory issues...etc). This way, we're ensured that the file doesn't end up in a corrupt state. It's slightly more expensive, but doesn't make a difference in this case.